packages/arb-token-bridge-ui/src/hooks/useNetworksAndSigners.tsx

```diff
@@ -190,9 +193,11 @@ export function NetworksAndSignersProvider(
 
       // Case 2: L2 is not supported by provider
       if (!providerSupportedL2.includes(_selectedL2ChainId)) {
-        history.replace({
-          pathname: '/'
+        // remove the l2chainId, keeping the rest of the params intact
+        updateQueryParams({
```

Comment:
This should be added to the dependency array of the `useEffect`? Make sure you install ESLint and double-check.
https://github.com/OffchainLabs/arb-token-bridge#development-tools

Reply:
I think you mean dependency array in `useCallback`? 
I have added it, but now to completely remove the warning there is a parameter `latestResult` also being added in the chain, however it's now being used in the callback.

Any reason why we have added this, and does it impact any use case? (why are we using `useLatest` anyway?)

![image](https://user-images.githubusercontent.com/7558499/191728317-960e015a-8df1-454a-a283-45812e7c9d11.png)